mla/time_profiles.py

```diff
@@ -238,13 +237,18 @@ def bounds(self, time_profile: GenericProfile) -> List[List[float]]:
             profile.
         """
 
-        if None in time_profile.get_range():
-            return [time_profile.get_range(), (0, None)]
+        if None in time_profile.range:
+            return [time_profile.range, (0, None)]
 
-        diff = time_profile.get_range()[1] - time_profile.get_range()[0]
+        diff = time_profile.range[1] - time_profile.range[0]
 
-        return [time_profile.get_range(), (0, diff)]
+        return [time_profile.range, (0, diff)]
 
+    @property
+    def range(self) -> Tuple[Optional[float], Optional[float]]: # Python 3.9 bug... pylint: disable=unsubscriptable-object
+        """Returns the min/max values for the distribution."""
```

Comment:
autodoc won't like this docstring. It will pull from the parent. No need for it here.